src/main/scala/diplomaticobjectmodel/model/OMPorts.scala

```diff
@@ -124,21 +124,26 @@ case class SystemPort(
   _types: Seq[String] = Seq("SystemPort", "OutboundPort", "OMPort", "OMDevice", "OMComponent", "OMCompoundType")) extends OutboundPort
 
 object OMPortMaker {
-  val protocolSpecifications = Map[ProtocolType, String](
-    AHBProtocol -> "AMBA 3 AHB-Lite Protocol",
-    AXI4Protocol -> "AMBA 3 AXI4-Lite Protocol",
-    APBProtocol -> "AMBA 3 APB Protocol",
-    TLProtocol -> "TileLink specification"
-  )
+  val protocolSpecifications: (ProtocolType, SubProtocolType) => String = {
+   case (AHBProtocol, AHBLiteSubProtocol) => "AHB Lite Protocol"
+   case (AHBProtocol, AHBFullSubProtocol) => "AHB Full Protocol"
+   case (AXI4Protocol, AXI4SubProtocol) => "AXI Protocol"
+   case (AXI4Protocol, AXI4LiteSubProtocol) => "AXI Lite Protocol"
+   case (APBProtocol, APBSubProtocol) => "APB Protocol"
+   case (TLProtocol, TL_UHSubProtocol) => "TileLink specification"
```

Comment:
why is this called "specification" when the others are called "Protocol"?

Reply:
Good point - I haven't the description for TL, but I can update it - do you think mentioning UL, UH and CS will also be a good idea?

Reply:
for that I'd defer to @hcook or @terpstra